{RepositoryName}/src/{Namespace}.Web/module.manifest

```diff
@@ -21,14 +21,6 @@
     <assemblyFile>{Namespace}.Web.dll</assemblyFile>
```

Comment:
VirtoCommerce.Core and VirtoCommerce.Platform.Core are not the same. Comment out 
`<dependency id="VirtoCommerce.Core" version="{CoreModuleVersion}" />`

Reply:
fixed